UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -230,6 +233,12 @@ void Awake()
             InitializeEnvironment();
         }
 
+        private void OnDestroy()
```

Comment:
In the develop branch, the method OnDestroy is used to reset some physics parameters, can you move DestroyAction() to this new method ?

Reply:
These changes have been made.